src/Filesystem/Replicate.php

```diff
@@ -25,13 +25,20 @@ class Replicate implements AdapterInterface, MetadataSupporter
 {
     /**
      * @var AdapterInterface
+     *
+     * @deprecated use $primary instead
      */
     protected $master;
 
     /**
      * @var AdapterInterface
      */
-    protected $slave;
+    protected $primary;
```

Comment:
Since this class is marked as final, I think we should make these new properties private.

Reply:
Yes